### ISSUE TITLE
Missing ray_trainer file referenced by main_ppo.py

### ISSUE THREAD

**Issue by itongggg** [ZJU-REAL/cooper#2]
Hi! Thanks for open-sourcing this project.

While trying to run the code, I hit an import error in **`main_ppo.py` line 18**:

```python
from .ray_trainer import RayPPOTrainer
```
However, I can’t find a corresponding file/module in the repository

**Comment by hearthht** [comment]
@itongggg  Hi~ Thanks for your interest in our work!
You’re right — the file **`ray_trainer.py`** was missing in the earlier version, which caused the import error in **`main_ppo.py`**. I’ve just uploaded the missing file — **`recipe/cooper/ray_trainer.py`**, so pulling the latest version of the repository.  I'm very sorry that my negligence caused this problem. Thank you very much for your correction！
If you have any other questions, feel free to reach out to us~

**Comment by itongggg** [comment]
@hearthht  Thanks for the quick fix!

**Comment by itongggg** [comment]
One quick question: in the paper you mention the reward model is trained jointly with the policy, but I couldn’t find the reward-model training step in recipe/cooper/ray_trainer.py. Did I miss where that’s implemented (or is it handled elsewhere)?
